test/integration/render-tests/fill-extrusion-pattern/@2x/style.json

```diff
@@ -3,6 +3,7 @@
   "metadata": {
     "test": {
       "height": 256,
+      "allowed": 0.03,
```

Comment:
This increase is quite high, we're at risk of never looking at these increase threshold again if we increase it by such a high value, it would be easier to pick that up again if we move it to platform todo instead.

Since the test might not ever be the same between platform (I imagine that we may have differences with the hardware implementation), that's that the kind of test that would benefit from having a per-platform image baseline to compare against, such as `expected-platform.png`. Since that's probably out of scope of this PR, I recommend moving that in a todo instead of increasing the threshold, so we ensure that it's picked up in a separate iteration of that work.

There might be other tests that fall in this category.

Reply:
Good point! I updated this value because the tests were visually very similar, but I agree that a value so high could fail to catch a regression. I'll remove/lower the "allowed" and add it to the TODOs.